packages/fuel-indexer-types/src/abi.rs

```diff
@@ -155,3 +189,17 @@ impl NativeFuelType for Call {
         type_id(FUEL_TYPES_NAMESPACE, "Call") as usize
     }
 }
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct Revert {
+    pub contract_id: ContractId,
+    pub ra: u64,
```

Comment:
What if we changed this to something like "error_val" in order to give users a more descriptive name? It's technically `ra` for "register A" but it would be nice to communicate the intended usage.